content/12.privacy.md

```diff
@@ -1,13 +1,15 @@
 ## Tip 10: Don't share models trained on sensitive data {#privacy}
 
 Practitioners may encounter datasets that cannot be shared, such as ones for which there would be significant ethical or legal issues associated with release [@doi:10.1371/journal.pcbi.1005399].
-Examples of such data include classified, confidential, trade secret biological data as well as medical records, certain genomic assays, and personally identifiable information [@doi:10.1038/s41576-020-0257-5].
-One of the greatest opportunities for deep learning in biology is the ability for these techniques to extract information that cannot readily be captured by traditional methods [@arxiv:1509.09292].
-The representation learning of the deep learning models can capture information-rich abstractions of multiple features of the data during the training process.
-However, these features may be more prone to leak the data that they were trained over if the model is shared or allowed to be queried with arbitrary inputs.
-Thus, with both deep learning and certain traditional machine learning methods (for example, _k_-nearest neighbors models, which learn by memorizing the full training data), it is imperative not to share models trained on sensitive data.
+Examples of such data include classified or confidential data, biological data related to trade secrets (_e.g._ genomic assay data), medical records, and personally identifiable information [@doi:10.1038/s41576-020-0257-5].
```

Comment:
```suggestion
Examples of such data include classified or confidential data, biological data related to trade secrets (for example, genomic assay data), medical records, and personally identifiable information [@doi:10.1038/s41576-020-0257-5].
```

Reply:
I'm not sure why genomic data would be trade secrets? I thought more of the concern was around leaking PII

Reply:
I'm thinking things like custom plasmids from synthetic biology, etc. Not sure why it would be genomic assay data.